content/misc/products-with-variants.md

```diff
@@ -0,0 +1,187 @@
+
+To tackle this "products with variants" problematic, we also provided an extension called **"Inner Variation Bundle"**. But its settings were pretty **complex**. And the fact that it was delivered in a separate bundle was really weird since this subject is at the really heart of the modeling of your products catalog.
+
+As you can see, the answers we gave at this time were not ideal.
+So we decided to do something about it. We have so many users that encounter such problematic that we had to react.
+
+As a result, we totally refounded the way we model the product with variants. In the 2.0 version of the PIM, it's now **easier** for our end-users to handle such products. This is a **real time saver**. They can see their variant products inside the grid, make smart search on them, they are also able to enrich the variant products almost like they were standard product.
+
+# So what's changed?
+
+The JSON standard format of a variant product is, in 2.0, the following:
+```json
+{
+  "identifier": "1111111195",
+  "family": "clothing",
+  "parent": "jack_brown",
+  "categories": [
+    "tshirts"
+  ],
+  "enabled": true,
+  "values": {
+    "ean": [
+      {
+        "locale": null,
+        "scope": null,
+        "data": "1234567890207"
+      }
+    ],
+    "size": [
+      {
+        "locale": null,
+        "scope": null,
+        "data": "s"
+      }
+    ],
+    "weight": [
+      {
+        "locale": null,
+        "scope": null,
+        "data": {
+          "amount": "800.0000",
+          "unit": "GRAM"
+        }
+      }
+    ],
+    "color": [
+      {
+        "locale": null,
+        "scope": null,
+        "data": "brown"
+      }
+    ],
+    "name": [
+      {
+        "locale": null,
+        "scope": null,
+        "data": "jack"
+      }
+    ],
+    "erp_name": [
+      {
+        "locale": "en_US",
+        "scope": null,
+        "data": "Jack"
+      }
+    ],
+    "collection": [
+      {
+        "locale": null,
+        "scope": null,
+        "data": [
+          "summer_2017"
+        ]
+      }
+    ]
+  },
+  "created": "2017-10-05T11:25:48+02:00",
+  "updated": "2017-10-05T11:25:48+02:00",
+  "associations": {},
+  "metadata": {
+    "workflow_status": "working_copy"
+  }
+}
+```
+
+As you can see, in the example above, there is no `variant_group` field anymore. So, how to migrate to 2.0, you will ask.
+
+It depends on your products catalog structure for these products with variants.
+
+## My variant groups are very structured
+
+You used the variant groups is a very effective way and they are really structured:
+ - You have less than 5 axes in your variant group,
+ - The products inside your variant group use the same family,
+ - The products inside your variant group use the same attributes.
+
+It's very good news for you! The solution is to use the new product model and family variant entities, that were introduced in the remodeling of the products with variants for the 2.0.
```

Comment:
maybe you can add a link to the API reference for product model & family variant endpoints?